main.c

```diff
@@ -107,6 +109,7 @@ int main()
     CALC_STATE_NUM(state_num);
     init_td_agent(&agent, state_num, 'O');
     load_model(&agent, state_num, MODEL_NAME);
+#elif defined(USE_MCTS)
```

Comment:
Would you put some initialization routines here?

Reply:
A routine for initializing MCTS is not required.

Reply:
> A routine for initializing MCTS is not required.

Append the above comment in this Preprocessor directive, so that the code looks more consistent.